### PR TITLE
handler_settings helper method

### PR DESCRIPTION
**Pull request by fessyfoo**
this is just a small refactoring.

DRY up fetching per handler settings from sensu's settings hash.

### CODE REVIEW

File: files/base.rb [hunk 1]
```diff
@@ -214,5 +214,13 @@ def power_of_two?(x)
     x==1
   end
 
+  def settings_Key
```
Comment (solarkennedy) [inline files/base.rb]: This is just a small nit, but did you mean to capitalize the K?

**Comment by solarkennedy** [comment]
Cool!

File: files/base.rb (hunk 1 above)
Comment (fessyfoo) [inline files/base.rb]: definitely did not.   will fix.

**Comment by fessyfoo** [comment]
fixed silly casing issue.

**Comment by solarkennedy** [comment]
Thanks!

**Pull request merged**